oauth2/authorize.php

```diff
@@ -32,9 +32,12 @@
 use OpenEMR\Common\Csrf\CsrfUtils;
 use OpenEMR\Common\Session\SessionUtil;
 use OpenEMR\RestControllers\AuthorizationController;
+use OpenEMR\Common\Logging\SystemLogger;
```

Comment:
we generally alphabetize the use stuff
(also think will likely get a PSR12 hit if the empty line at beginning and end of the user block)

btw, @sjpadgett is right; i am a huge fan of your error/debug and tests stuff :)

Reply:
Fixed.